R/design_helper_functions.R

```diff
@@ -248,7 +248,7 @@ get_estimands_single_design <- function(design) {
 
 #' Obtain the preferred citation for a design
 #'
-#' @param design a design object created by \code{declare_design}
+#' @param design a design object created using the + operator
```

Comment:
this is awkward phrasing, what does hadley use in ggplot2 docs?

Also can use inheritparams to make sure its consistent.